ratlib/autocorrect.py

```diff
@@ -15,15 +15,16 @@
 
 class CorrectionResult:
     # Table of lookalikes
-    lookalikes = {'L': dict(zip("01258", "oizsb"))}
-    lookalikes['D'] = dict((v, k) for k, v in lookalikes['L'].items())
+    digits = "001258"
+    letters = "doizsb"
+    lookalikes = {'L': dict(zip(digits, letters)), 'D': dict(zip(letters, digits))}
 
     # Allowed characters for each pattern.  Matches regex syntax within a [group]
     allowed = {'L': 'a-z', 'D': '0-9'}
 
     # Create the regular expression we use to match systems
     pattern = (r'\w+\s+(?P<l>LL-L\s+L)(?P<d>D+(?:-D+)?'
-               r')\b')  #  L and D will be replaced from lookalikes and alowed.
+               r')\b')  # L and D will be replaced from lookalikes and allowed.
```

Comment:
please undo this modification, there is by PEP8 supposed to be two spaces after `#` for inline comments.

Reply:
Inline comments should start with '# ' according to PEP8 
https://www.python.org/dev/peps/pep-0008/#inline-comments

Reply:
it appears i misread the part `Inline comments should be separated by at least two spaces from the statement. `

in either case, this is an out-of-scope change.

Reply:
We're not discussing PEP8 stuff for this repo, it's getting replaced anyway.